code/modules/projectiles/projectile/beams.dm

```diff
@@ -62,6 +62,17 @@
 	impact_effect_type = /obj/effect/overlay/temp/impact_effect/blue_laser
 	light_color = LIGHT_COLOR_BLUE
 
+/obj/item/projectile/beam/m_disabler
+	name = "musket beam"
+	icon_state = "ice_2"
+	damage = 48 //three shots to down someone

```

Comment:
A piece of trash you piece together should not be superior to the proper ones in any manner.